client/app/utils/transitImmutableConverter.js

```diff
@@ -16,6 +16,10 @@ const toImage = (data) => {
   return styles;
 };
 
+// Outside of this file we should only pass UUID references, no need to export
+const UUID = Immutable.Record({ stringRep: '' });
+const toUUID = (transitUuid) => new UUID({ stringRep: transitUuid.toString() });
```

Comment:
That way we don't lose the type information: it's not a string, it's an UUID. It pretty much only matters when we're serialising the data back to Transit, but parsing it separately everywhere would be a hassle. `ListingModel`'s id is parsed by this same function, so no need to do it separately.

I'll change the field name though, it's not good